pizza-recipes.md

```diff
@@ -174,3 +174,78 @@ Embrace this blend of flavors as you forgo store-brought sauce for this nuturiou
 6. Cook for 30 mins, adjust water if thick.
 
 7. Let it cool. Your pizza sauce is ready to use.
+## Nigeria pizza
+![Nigeria-suya-pizza](images/nigeria-suya-pizza.jpg)
+
+The first experience with this snack might not be interesting. It may taste strange for your first time especially for those with Nigeria taste bud.lol! But not to worry with time you will get used to taste and will start running back for more of it.
+The interesting thing about these snacks is any food item can be used as its topping so far as the combo pleases you.
+
+## Ingredients
+The following ingredients are neeeded for a 12-inch diameter pizza.
+
+*For the pizza dough*
+The following quantities of ingredients make a chunky pizza crust. If you want a slender pizza crust but still 12 inches diameter, use half the quantities of ingredients.
+
+- 300g plain flour 
+- 3g or 2 teaspoons yeast
+- Four tablespoons olive oil
+- 150ml of warm water
+- One teaspoon of salt
+- One teaspoon of sugar
+- Margarine (for the oven tray)
+
+*For the toppings*
+- 250g Mozzarella cheese
+- Four table spoons of Tomato stew
+- Two mushrooms
+- One tablespoon of minced/ground meat
+- Red bell pepper
+- Green bell pepper
+- Two tablespoons of olive oil
+
+## Instructions
+- Mix the pizza dough
+
+1. Sieve the plain flour into a clean dry bowl. The bowl should be large enough for mixing the flour and other ingredients. Why it is required to sift the flour is because it airs the flour and breaks up any available lumps making it easier to mix.
+2. Add the yeast to the dry flour and stir to mix. This is important so the salt which is added next does not touch the yeast.
+3. Put in the sugar and salt and mix thoroughly
+4. Then add the olive oil and mix thoroughly
+5. Begin adding the warm water and mix up with a spatula at the same time till all the water has been engrossed. Then continue by hand till even soft dough is achieved. Fold up the dough and lay it in the bowl.
+6. Cover the bowl with a thin plastic film/warm towel and place in a warm place for an hour for the dough to rise. You can place it in a turned off oven that can serve as a warm place.
+
+*Prepare the oven tray*
+Usually, Pizza is laid on a pizza stone and baked in a clay oven, there are mostly used by those living but unless you live in a beautiful Italian countryside, what most of us have is a standard kitchen oven.
+And you most probably do not have a pizza stone.
+The Pizza will be placed in an oven tray and bake it in a standard kitchen oven.
```

Comment:
[suggestion] While the mentioning of an Italian countryside is a great attempt at humor, it might offend others as it implies that not being in that location makes the pizza inauthentic, so I suggest keeping this direction more general.
```suggestion
## Prepare the oven tray
Place the pizza in an oven tray. If you happen to have a pizza stone, feel free to use that instead. 
```